docs/patterns/components/Modal/index.js

```diff
@@ -144,6 +146,55 @@ class Demo extends Component {
 }
           `}
         </Code>
+        <div className="Demo-props">
+          <h2>Props</h2>
+          <PropDocs
+            docs={{
+              children,
+              className,
+              variant: {
+                type: 'string',
+                description: 'The style of Modal to display.',
+                default: 'default'
+              },
+              heading: {
+                type: 'React.ReactElement<any> or object',
+                description:
+                  'Displayed in the heading at the top of the Modal. Optional to pass heading level.',
+                required: true
+              },
+              onClose: {
+                type: 'function',
+                description: 'Function called when the Modal is closed'
+              },
+              portal: {
+                type: 'any',
+                description: 'The parent element to render the Modal within.',
+                default: 'document.body'
+              },
+              show: {
+                type: 'boolean',
+                description: 'Whether or not to show the Modal'
+              },
+              dialogRef: {
+                type: 'function or function.current',
+                description: 'Pass a ref to the Modal.'
+              },
+              forceAction: {
+                type: 'boolean',
+                description:
+                  'If true, no close button will be rendered and the user must take action with Modal content.',
+                default: 'false'
+              },
```

Comment:
```suggestion
```

This isn't true, modals are implicitly `forceAction={false}`: 
https://github.com/dequelabs/cauldron/blob/fa8e54c533a55ae2c5164802675a6a3fbf8fa985/packages/react/src/components/Modal/index.tsx#L15

Reply:
🤔 I have default as false - is the description confusing?

Reply:
`forceAction` isn't a valid prop for the modal component, see the interface:

https://github.com/dequelabs/cauldron/blob/3ba9dc2a80bf52567c00361f9bddbe4ab6afa154/packages/react/src/components/Modal/index.tsx#L5-L7

We omit the `forceAction` prop, so we don't need to include that property for documentation.